### ISSUE TITLE
fx-compat: Wooshy doesn't work as well in Info pane

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#2736]
Doesn't find info pane elements at all, whereas it did in Zotero 6. But even there, matching the labels wasn't actually useful, because clicking them didn't open the field, so we should probably make that work. (And then we should change the URL row to use a launch icon like we do in the web library, which is probably better anyway.)

(Have to disable [these lines](https://github.com/zotero/zotero-standalone-build/blob/10bd65a21cac1deb4e610cf8ed78189980b8ed33/build.sh#L380-L382) to test in a source build until Wooshy adds our additional bundle identifiers.)

**Comment by adomasven** [comment]
Related https://github.com/zotero/zotero/issues/1956

**Comment by dstillman** [comment]
Closing per https://github.com/godbout/Wooshy.docs/issues/25#issuecomment-2558429593
